website/views.py

```diff
@@ -329,6 +330,15 @@ def resolve_guid(guid, suffix=None):
 
             return send_from_directory(ember_osf_web_dir, 'index.html')
 
+        if isinstance(referent, Registration) and not suffix:
+            if waffle.flag_is_active(request, 'ember_registries_detail_page'):
+                # Route only the base detail view to ember
```

Comment:
Question for CR: sub-routes of retractions redirect to the base `/[guid]/` route. Will registries be able to handle this immediately?